eva-accession-pipeline/README.md

```diff
@@ -0,0 +1,76 @@
+# EVA accession pipeline
+
+This module can be used to assign SubSNP (SS) accessions to a set of variants from a VCF file.
```

Comment:
Should we also state that only EVA currently has the authority to issue accessions for non-human species and link to the relevant NCBI/EVA news release?